catlearn/algebra_models.py

```diff
@@ -124,6 +125,59 @@ def comp(self, data1: torch.Tensor, data2: torch.Tensor) -> torch.Tensor:
         return data1 + data2
 
 
+class VectMultAlgebra(Algebra):
```

Comment:
I am confused. Those parts were included in Igor' pull request as well. IOs this your new code? Or simply not rebased? Or what?

Reply:
Actually that's because Igor rebased on an early version of this branch